include/rocksdb/write_batch.h

```diff
@@ -213,6 +213,9 @@ class WriteBatch : public WriteBatchBase {
   // Otherwise returns Status::OK().
   Status PopSavePoint() override;
 
+  // Append a batch to the end.
+  Status Append(const WriteBatch* batch, const bool WAL_only = false);
```

Comment:
Need to comment on what `WAL_only` means.

Also, how do you compare with the interface of WriteBatch::Append() and a separate utility function? It feels to me that if it is not among the core functionality of write batch, it might be easier to leave it outside WriteBatch.

Reply:
hi @siying do you wish we provide a utility function `WriteBatchAppend` which is the wrapper of `WriteBatchInternal::Append`?

Reply:
I mean something like
```
Status AppendWriteBatch(WriteBatch& write_batch, const WriteBatch& write_batch_to_append, bool wal_only);
```